CHANGES.md

```diff
@@ -69,8 +69,10 @@ To be released.
 
 ### Added interfaces
 
- -  Added `BlockChain<T>.TipChanged` event handler which invoked with argument
-    of `long` typed tip index when `BlockChain<T>.Tip` is changed.  [[#517]]
+ -  Added `BlockChain<T>.TipChanged` event which is invoked with an argument
+    of `BlockChain<T>.TipChangedEventArgs` when `BlockChain<T>.Tip` is changed.
+    [[#517], [#526]]
+ -  Added `BlockChain<T>.TipChangedEventArgs` class.  [[#517]]
```

Comment:
Didn't TipChangedEventArgs be added in #526?
```suggestion
 -  Added `BlockChain<T>.TipChangedEventArgs` class.  [[#526]]
```

Reply:
👀 You're right. I'll amend.